src/main/java/frc/robot/subsystems/SuperStructure.java

```diff
@@ -22,75 +22,127 @@ public SuperStructure(
         spinnyClaw.holdExtakePower().withTimeout(0.15),
         stow());
   }
 
   public Command levelTwo(BooleanSupplier score) {
     return Commands.sequence(
         Commands.parallel(
-            elevator.setLevel(ElevatorSubsystem.LEVEL_TWO_PRE_POS),
-            armPivot.moveToPosition(ArmPivot.PRESET_UP),
+            elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_TWO_PRE_POS),
+            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_UP),
             spinnyClaw.stop()),
-        armPivot.moveToPosition(ArmPivot.PRESET_L2_L3),
+        armPivot.moveToPosition(ArmPivot.CORAL_PRESET_L2_L3),
         Commands.waitUntil(score),
-        elevator.setLevel(ElevatorSubsystem.LEVEL_TWO_POS),
+        elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_TWO_POS),
         spinnyClaw.holdExtakePower().withTimeout(0.15),
         stow());
   }
 
   public Command levelOne(BooleanSupplier score) {
     return Commands.sequence(
         Commands.parallel(
-            elevator.setLevel(ElevatorSubsystem.LEVEL_ONE_POS),
-            armPivot.moveToPosition(ArmPivot.PRESET_L1),
+            elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_ONE_POS),
+            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_L1),
             spinnyClaw.stop()),
         Commands.waitUntil(score),
-        elevator.setLevel(ElevatorSubsystem.LEVEL_TWO_POS),
+        elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_TWO_POS),
         spinnyClaw.holdExtakePower().withTimeout(0.15),
         stow());
   }
 
   public Command stow() {
     return Commands.parallel(
-        elevator.setLevel(ElevatorSubsystem.STOWED),
-        armPivot.moveToPosition(ArmPivot.PRESET_STOWED),
+        elevator.setLevel(ElevatorSubsystem.CORAL_STOWED),
+        armPivot.moveToPosition(ArmPivot.CORAL_PRESET_STOWED),
         spinnyClaw.stop());
   }
 
   public Command intake() {
     return Commands.sequence(
             Commands.parallel(
-                elevator.setLevel(ElevatorSubsystem.PRE_INTAKE),
-                armPivot.moveToPosition(ArmPivot.PRESET_DOWN),
+                elevator.setLevel(ElevatorSubsystem.CORAL_PRE_INTAKE),
+                armPivot.moveToPosition(ArmPivot.CORAL_PRESET_DOWN),
                 spinnyClaw.intakePower()),
             // Commands.waitUntil(armSensor.inTrough()),
-            elevator.setLevel(ElevatorSubsystem.INTAKE),
+            elevator.setLevel(ElevatorSubsystem.CORAL_INTAKE_POS),
             Commands.waitSeconds(0.1),
             spinnyClaw.stop(),
-            elevator.setLevel(ElevatorSubsystem.PRE_INTAKE),
+            elevator.setLevel(ElevatorSubsystem.CORAL_PRE_INTAKE),
             stow())
         .withName("Intake");
   }
+
+  public Command algaeLevelThreeFourIntake() {
+    return Commands.sequence(
+        Commands.parallel(
+            spinnyClaw.algaeIntakePower(),
+            armPivot.moveToPosition(ArmPivot.ALGAE_REMOVE),
+            elevator.setLevel(ElevatorSubsystem.ALGAE_LEVEL_THREE_FOUR)),
+        algaeStow());
```

Comment:
Theoretically speaking there should be a wait here and in `algaeLevelTwoThreeIntake()` so we don't immediately stow after reaching the intake position, but since we don't have algae intake mechanically right now it doesn't really matter. (You can just file this in your memory and resolve this conversation)